setup.cfg

```diff
@@ -11,4 +11,11 @@ output_file = server.pot
 mapping_file = babel.cfg
 
 [nosetests]
-exclude-dir = prod_api/tests/
\ No newline at end of file
+exclude-dir = prod_api/tests/
+
+[mypy]
+python_version = 3.6
```

Comment:
ok so we ditch 3.5 for good then?

Reply:
I've put 3.6 because Travis does the check and it uses 3.6

Reply:
It was discussed in the core group, Superdesk v2.0 will drop support for Python <= 3.5